src/components/molecules/task/task.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { Grid, Dropdown, Checkbox } from "semantic-ui-react";
+import {
+  StyledGridElipsisColumn,
+  StyledGridCheckboxColumn,
+  StyledGridTaskColumn,
+} from "./style/task.styled.component";
+
+export type ITask = {
+  taskName: String;
+  isTask?: boolean;
+  taskID: number;
+  onHandleDropDownChange: (e: any, data: any, taskID: number) => any;
+  onHandleCheckboxChange: (e: any, data: any) => any;
+};
+
+const TaskComponent = ({
+  taskName = "My Task",
+  isTask = false,
+  taskID = 0,
+  onHandleDropDownChange,
+  onHandleCheckboxChange,
+}: ITask) => {
+  return (
+    <>
+      <Grid.Row>
+        {!isTask ? (
+          <StyledGridCheckboxColumn width={1}>
+            <Checkbox
+              value={taskID}
+              name="idTask"
+              onChange={(e, data) =>
+                onHandleCheckboxChange ? onHandleCheckboxChange(e, data) : {}
```

Comment:
what if we just add a default value for this prop instead of adding extra conditions here?